### ISSUE TITLE
osteoderms

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2423]
There has been some effort recently to tighten up the terminology used for dermal bones. Right now we use both osteoderm and scute in the database. Scute is particularly problematic because it has been used as a catch-all term for both bony and keratinous structures in the skin as well as turtle shell which science has not decided is dermal bone or not.

My recommendation is to have part osteoderm (definition: dermally derived bone emplaced in the dermis) with the part attribute: osteoderm morphology (definition: terms differentiating osteoderms by shape and size) including ossicle, "scute, bony", spine, spike, and plate.

_Originally posted by @Nicole-Ridgwell-NMMNHS in https://github.com/ArctosDB/data-migration/issues/138#issuecomment-548523119_

**Comment by Jegelewicz** [comment]
Before I set this up - I'd like to see if anyone else has thoughts on this.

**Comment by dustymc** [comment]
I can't say anything useful about the specifics, but I think the trend of a general part name optionally clarified by specific attributes is a huge step in the right direction regarding discoverability. People who just want bumpy-things can find them without tracking down the maybe dozen of variants, people who want THAT bumpy thing can find it and it alone, yay everybody.

**Comment by Jegelewicz** [comment]
@mbprondzinski @dperriguey @jtgiermakowski @mvzhuang @atrox10 thoughts or comments before I proceed to set this up?

**Comment by mbprondzinski** [comment]
https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/scute

Just to add fuel to the fire...

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
> Just to add fuel to the fire... Sometimes I think Arctos way overthinks the searchability factor.

It doesn't make sense to me to have to search under several different part names to find bony bits of armor.

**Comment by dustymc** [comment]
>search under several different part names to find bony bits of armor.

That's the point of my suggestion (mostly stolen from @Jegelewicz  - yay "the community"!) regarding part attributes (which could almost certainly use some UI refinement). Users who want some sort of bony-bit can find it by searching the one (eventually) part which encompasses all bony-bits. Users who want THAT bony-bit (and trust us and buy into our definitions and etc.) can add a "sub-part" parameter to get what they want and nothing else.

The same pattern could apply to a LOT of parts, and perhaps even be much more general than current values without losing any resolution. I'd love to see the 'first step' part vocabulary be perhaps a couple dozen terms with all the refinements a step further in; I suspect most users want skulls, skins, tissues, appendicular bones, etc., and with more extensive use of part attributes we might be able to get there while still allowing the occasional focused user to find the third left distal tarsal.

tl;dr: Agree, but that doesn't mean we have to lose information.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I totally agree, was just trying to understand @mbprondzinski 's comment.

**Comment by Jegelewicz** [comment]
@Jegelewicz create code table for osteoderm type

Turn all parts with those names into osteoderm and add appropriate type.

**Comment by Jegelewicz** [comment]
Code Table Committee discussed this today and agreed that it would be good to create a new part attribute:

**osteoderm morphology** = terms differentiating osteoderms by shape and size.

with a code table to control the types

ctosteoderm

with the following values:

term | definition 
 -- | -- 
ossicle | 
scute, bony | 
spine | 
spike |  
plate | https://en.wikipedia.org/wiki/Plate_(anatomy)#Flat_bones_(examples:_bony_plates,_dermal_plates)_of_vertebrates

@Nicole-Ridgwell-NMMNHS can you fill in missing definitions?

After this is set up, the following parts could be converted to "osteoderm" with the associated part attribute: scute, spine, however this is going to need manual review as both may refer to non-bony structures (porcupine quills, keratinous covering of turtle osteoderms).

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
ossicle: a very small osteoderm element, generally smaller than 1 cm

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
scute, bony: non-specific osteoderm, usually used to describe osteoderms that are not ossicles, spines, spikes, or plates. Do not use for keratinous scutes.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
spine: an osteoderm with a narrow, spine-like projection
spike: an osteoderm with a tall, conical projection

**Comment by Jegelewicz** [comment]
Code Table Committee discussed this today and agreed that it would be good to create a new part attribute:

**osteoderm morphology** = terms differentiating osteoderms by shape and size.

with a code table to control the types

ctosteoderm

with the following values:

term | definition 
 -- | -- 
ossicle | a very small osteoderm element, generally smaller than 1 cm
scute, bony | non-specific osteoderm, usually used to describe osteoderms that are not ossicles, spines, spikes, or plates. Do not use for keratinous scutes.
spine | an osteoderm with a narrow, spine-like projection
spike |  an osteoderm with a tall, conical projection
plate | https://en.wikipedia.org/wiki/Plate_(anatomy)#Flat_bones_(examples:_bony_plates,_dermal_plates)_of_vertebrates 

After this is set up, the following parts could be converted to "osteoderm" with the associated part attribute: scute, spine, however this is going to need manual review as both may refer to non-bony structures (porcupine quills, keratinous covering of turtle osteoderms). Definitions for those parts will also need review.

**Comment by Jegelewicz** [comment]
Given #3710 I suggest we add these to "anatomical direction" once we decide what that table should really be called. Also part of #3708

**Comment by Jegelewicz** [comment]
As part of #3708 can we get this ball rolling? First step add these to the part_modifier table BUT where the heck is it?

![image](https://user-images.githubusercontent.com/5725767/144635252-ae222a18-bc92-47a2-aa34-2df98fe5075f.png)

@dustymc ?

**Comment by Jegelewicz** [comment]
OK, the terms [here](https://github.com/ArctosDB/arctos/issues/2423#issuecomment-677931471) are added to the part modifier table except for scute, bony.

Here's why I haven't added it - An unspecified osteoderm just shouldn't have a modifier and I dislike the idea of the comma thing just working it's way down the code table ladder. I thought about adding simply scute with the definition = A non-specific osteoderm, usually used to describe osteoderms that are not ossicles, spines, spikes, or plates, but scute is already a part name specifically assigned to keratinous stuff. Or so I thought. The current definition of scute makes it a partial synonym for osteoderm.

scute - A bony external plate or scale overlaid with horn, as on the shell of a turtle, the skin of crocodilians, and the feet of birds. The term is also used to describe the anterior portion of the mesonotum in insects as well as some arachnids (e.g., the family Ixodidae, the scale ticks). Wikipedia

osteoderm - Bone deposits found in the skin, including bone scales, scutes, ridges, and plates. Wikipedia

I think we need to work this out so that we are clear about what is a scute and what is an osteoderm IN ARCTOS

@Nicole-Ridgwell-NMMNHS @jtgiermakowski @hkevans @ebraker @aklompma thoughts/ideas?

**Comment by Jegelewicz** [comment]
FWIW from Wikipedia

Osteoderms are bony deposits forming scales, plates, or other structures based in the dermis https://en.wikipedia.org/wiki/Osteoderm

and the definition we are currently using for scute comes directly from Wikipedia.

But I think this will help us create a better set of definitions?

The term osteoderm literally means bones in the skin and that's exactly what these are, piece of bony armour that sit in the skin. Scutes on the other hand are the keratinous sheathes that cover the osteoderms. - https://archosaurmusings.wordpress.com/2011/05/08/scutes-and-osteoderms/

**Comment by dustymc** [comment]
>IN ARCTOS

I'm mostly just here to agree with that - in general, I think we should be forcing requestors to come up with much better definitions, and then treating changes to those definitions in the same way we treat new terms (because those actions are functionally identical). "Outside" usage should certainly shape those terms and definitions, but eg wikipedia is almost never quite what's needed to consistently enter data (which provides the path to consistently finding data).

"bony" seems inherent in "scute" - I think the latter is all that's necessary.

I think https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#scute should go away as part of this, so I don't think there's any potential conflict there. That should be a new Issue, here's who's using:

```
 guid_prefix | nrecs 
-------------+-------
 ALMNH:Paleo |    43
 NCWRC:Herp  |     1
 NMMNH:Paleo |  1847
 UTEP:ES     |    83
```

Homonyms are just incompatible with normalization - "insects as well as some arachnids" will just have to find their own term if that becomes an issue. That might be important in the definition, but it should be prefixed with "do not use for...."

**Comment by Jegelewicz** [comment]
@dustymc I don't think scute can go away - I know that UTEP has some of the keratin plates from a turtle without any other stuff - those are simply scutes....

**Comment by dustymc** [comment]
Even with good definitions, having one string in two part-tables doesn't seem ideal.

Maybe we're still trying to be too specific, is there some fancy word for "skin-bones" (some subset of https://en.wikipedia.org/wiki/Dermal_bone??)

Maybe just "bony scute"?

What's the current part used for? I'm guessing a little this, little that...


<img width="461" alt="Screen Shot 2021-12-03 at 3 48 23 PM" src="https://user-images.githubusercontent.com/5720791/144686876-b2266089-6efc-4c33-8b57-2ee93b74dd01.png">

**Comment by Jegelewicz** [comment]
As @Nicole-Ridgwell-NMMNHS said, the cleanup for scute will require eyeballs. For example [this one](https://arctos.database.museum/guid/UTEP:ES:278-385), which might be the bony OR the keratinous part of a carapace or might include both!.

![image](https://user-images.githubusercontent.com/5725767/144687784-0181929f-dd1b-4e4d-aecf-cfe25ccf5950.png)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
> An unspecified osteoderm just shouldn't have a modifier

I think you are correct here. Leave "scute, bony" out of the parts modifier table. You have my permission to mass update the NMMNH:Paleo scutes to osteoderm with remark that scute was the original part name.

I'm sure there will be folks who still unknowingly use part scute incorrectly for bony 'scutes' rather than keratinous scutes, but that is why we have the definitions.

**Comment by Jegelewicz** [comment]
> You have my permission to mass update the NMMNH:Paleo scutes to osteoderm with remark that scute was the original part name.

@dustymc I was going to use the manage parts thing to do this, but some of these already have part remarks that I don't want to over-write. Any chance we could add the option to append a remark?

![image](https://user-images.githubusercontent.com/5725767/152562061-85a7b7b6-d1e7-4fb6-aa6f-a613882ef161.png)

if not, the option is to add new parts and delete the old - kinda a bummer, but do-able.

**Comment by Jegelewicz** [comment]
Reading up on this and here is what I think we should do about "bony scutes".

Instead of osteoderm with part modifiers of "scute" and "bony" these objects should just be unmodified osteoderms. I come to this conclusion for two reasons.

1. @Nicole-Ridgwell-NMMNHS said above "mass update the NMMNH:Paleo scutes to osteoderm"
2. Scutes with a bony base are properly called osteoderms. [Wikipedia](https://en.wikipedia.org/wiki/Scute)

The part name "scute" should be reserved for 

A bony external plate or scale overlaid with horn, as on the shell of a turtle, the skin of crocodilians, and the feet of birds. **If the horn or keratinous layer is not present, the appropriate part name is osteoderm.** The term is also used to describe the anterior portion of the mesonotum in insects as well as some arachnids (e.g., the family Ixodidae, the scale ticks). 

Sentence in bold is the addition to the current code table definition. I also considered adding:

**If both the keratinous and bony parts of the scute are present, it will be advantageous to record two parts, scute and osteoderm.**

But that may just be asking way too much or also means this isn't the right approach? 

FWIW here are definitions for bony and keratinous

bony - of or like bone. "the bony plates that protect turtles and tortoises"

keratinous - containing or made from keratin

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
> Instead of osteoderm with part modifiers of "scute" and "bony" these objects should just be unmodified osteoderms.

I went back and forth on this quite a bit when making the original request. Recently I think, at least in the paleo community, folks have moved toward just using osteoderm rather than scute for relatively flat but not tiny osteoderms. The original basis for this distinction is a bit unclear to me (for example read through the comments on this paleo blog post: [Scutes and osteoderms](https://archosaurmusings.wordpress.com/2011/05/08/scutes-and-osteoderms/#:~:text=The%20term%20osteoderm%20literally%20means,sheathes%20that%20cover%20the%20osteoderms.)), but it makes sense to disambiguate scute. Scute (for osteoderms) is still used though especially informally and is common in older publications.

> If both the keratinous and bony parts of the scute are present, it will be advantageous to record two parts, scute and osteoderm.

That sounds to me like it would be annoying, but someone with modern verts should weigh in.

**Comment by dustymc** [comment]
Done?

**Comment by Jegelewicz** [comment]
Nope - on Code Table Committee Agenda.

**Comment by Jegelewicz** [comment]
new definition for [scute](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#scute) part

A bony external plate or scale overlaid with horn, as on the shell of a turtle, the skin of crocodilians, and the feet of birds. If the horn or keratinous layer is not present, the appropriate part name is osteoderm along with the part modifier scute, bony. If both the keratinous and bony parts of the scute are present, it will be advantageous to record two parts, scute and osteoderm. The term is also used to describe the anterior portion of the mesonotum in insects as well as some arachnids (e.g., the family Ixodidae, the scale ticks).

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Code table committee says add osteoderm terms to part modifiers, definition for part scute will need to be updated. Collections will have to sort out on their own whether their scutes are bony or keratinous.

**Comment by Jegelewicz** [comment]
[osteoderm](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#osteoderm) definition revision to make it clear when to use scute instead.

Bone deposits found in the skin, including bone scales, scutes, ridges, and plates. <a target="_blank" class="external" href="https://en.wikipedia.org/wiki/Osteoderm">Wikipedia</a> If the horn or keratinous layer is present, the appropriate part name is scute. If both the keratinous and bony parts of the scute are present, it will be advantageous to record two parts, scute and osteoderm.

**Comment by Jegelewicz** [comment]
All terms added to code tables. Now need to fix NMMNH:Paleo parts.

**Comment by Jegelewicz** [comment]
Also check ALMNH:Paleo and UTEP:ES

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I should be able to use the parts table download with Bulk Edit Parts and Bulkload Part Attributes to get where I need to be. I was thinking about the turtles some more and about how recent research shows they are not osteoderms, so actually decided to first update all turtle scutes to part carapace with appropriate condition, remark, and part modifiers.

**Comment by Jegelewicz** [comment]
ALMNH:Paleo osteoderms all seem OK but scute parts need review by on-site personnel.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Still reviewing scute parts, but yeah this is fine to close.

**Comment by Jegelewicz** [comment]
Thanks!
